README.md

```diff
@@ -17,159 +17,28 @@ Additionally the file `data/model/Daylight Box UV.png` is by [KIIRA](https://ope
-
-## Ice/Antarctica or something
-
-- Pack ice
-- Collapsing cliffs
-
-## Canyon
-
-- Huge bridge
-- Dam with small lake
-- Just a small lake
-
-## British Canal
-
-- Small cities
-- Green mountains
-
-## Tropical beach
-
-- Palms
-- Beaches
-- Clear water
-- Cruise ship in water?
-- Dolphins?
-- Planes landing near water maybe?
-
-## Indonesia (maybe philipines?)
-
-- Rice fields
-- Temples
-- Cliffs
-
-## Jungle [maybe parts of indonesia?]
-
-- Trees on sides
-- Dead trees or rocks in center of water as obstacles
-
-## Miami
-
-- Races along beaches
-- Detour through city
-
-## Some industrial thing
-
-- Cranes on land
-- Freightships on canal towards harbour
-
-## Hometown lake
-
-- Lake surrounded by green mountains
-- Fisherboats and smaller piers near edges
-
-## Oriental
-
-- Waterparks and rollercoasters nearby [maybe one going over a canal?]
-
-
-# Resources
-
-## Hydroplanes and boating
-
-- Good read to get you started: http://www.hydrobowl.com/anatomy.htm
-- http://www.rcboatmag.com/select-the-right-hull/ also has a good overview
-- More about turn fins on hydroplanes: http://www.rcuniverse.com/forum/speed-rc-nitro-boats-117/10229607-hydroplane-without-turn-fin.html
-- https://www.youtube.com/watch?v=_AArltTNRaA
-- http://www.h1unlimited.com/hydros-101/
-
-## Sound effects
-
-- Reference videos: https://www.youtube.com/watch?v=fqXdg6wYHF4
-- About engine RPM to sound http://www.nogripracing.com/forum/showthread.php?p=1146183
-- Swoosh sound effect from https://www.youtube.com/watch?v=dIuf8s8Z2fM
-
-## Soundtrack
-
-Here is a collection of songs which might be acceptable.
-Need to figure out licensing / copyrights and such.
-
-It should mostly be some kind of electronic for short menu sequences.
-For the race part focus should be on punk, rock and metal.
-
-Songs will probably be downloaded from soundcloud or other streaming providers.
-
-- https://soundcloud.com/angry-beavers-cover-band/magic-key-one-t-cool-t-metal-cover < !
-- https://soundcloud.com/jimgryp/2-unlimited-jump-for-joy-2k16-jim-gryp-remix < menu?
-- https://soundcloud.com/marcokrasinski/tunz-tunz-no-limit-2unlimited < ?
-- https://soundcloud.com/mareik-40754131/mareik-universe-noise-milo-bice-curbi-51-pep-rash < menu music?!
-- https://soundcloud.com/newdamagerecords << goldmine!
-- https://soundcloud.com/noconnectionuk/07-rebel-song
-- https://soundcloud.com/nonpoint/divided-conquer-them < maybe has more songs?
-- https://soundcloud.com/peterpanspeedrock
-- https://soundcloud.com/thevines/02-ladybug << ?
-- https://www.youtube.com/watch?v=adnAIJUnyYg << ?????
-- https://www.youtube.com/watch?v=bIaQ9VsnHnU
-- https://www.youtube.com/watch?v=CwFmidRtQcw << Whole channel is awesome it seems!
-- https://www.youtube.com/watch?v=JmBbF8JXRBc
-- https://www.youtube.com/watch?v=-QA0WLSZqJk << Potentially, easy to get permission [hopefully :P]
-- https://www.youtube.com/watch?v=TqwZDZexRg0
-
+Now you can run the generated executable called `main`
```

Comment:
Thanks!

I'll try to find a better solution. I guess we should have cmake copy data files to the build folder automatically (but only the final (*.obj, *.png, ..) files, not project (*.blend, ..) files)